src/com/flowers/Main.java

```diff
@@ -0,0 +1,30 @@
+package com.flowers;
+
+import java.util.stream.Stream;
+
+public class Main {
+    private static String fileName = "C://Users/Admin/IdeaProjects/HomeWork06/src/com/flowers//testFlower.txt";
```

Comment:
Use constants. And don't use an absolute path. For example, on other PCs, it would not work as on your. 
See
https://www.mkyong.com/java/java-read-a-file-from-resources-folder/